### PR TITLE
Services & Contacts Page

### LINKED ISSUE #5: Contacts

### PR DESCRIPTION
**Pull request by bellblex**
Closes #5 & #8

### CODE REVIEW

**Review by thePeras — APPROVED** [review]
To be honest, "Send a message" forms text fields seems a little bit off. 
Also, I think "give your feedback" can be changed to "Give your anonymous feedback" 
I am approving, since not really important things for now

File: components/Contact/index.tsx [hunk 1]
```diff
@@ -65,38 +61,46 @@ const Contact = () => {
                 method="POST"
               >
                 <div className="mb-7.5 flex flex-col gap-7.5 lg:flex-row lg:justify-between lg:gap-14">
-                  <input
-                    type="text"
-                    placeholder="Full name"
-                    className="w-full border-b border-stroke bg-transparent pb-3.5 focus:border-waterloo focus:placeholder:text-black focus-visible:outline-none dark:border-strokedark dark:focus:border-manatee dark:focus:placeholder:text-white lg:w-1/2"
-                  />
+                  <div className="p-2 flex flex-col">
+                  <label htmlFor="name">Full name</label>
```
Comment (rubuy-74) [inline components/Contact/index.tsx:58]: These labels don't seem to be pointing at anything

File: components/Info/index.tsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
+"use client";
+
+import { motion } from "framer-motion";
+import Image from "next/image";
+import ScheduleTable from "../Table";
+
+const horariosData1 = [
```
Comment (rubuy-74) [inline components/Info/index.tsx]: The variables should be in English, for consistency

File: components/Info/index.tsx [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+                <span className="relative inline-block before:absolute before:bottom-2.5 before:left-0 before:-z-1 before:h-3 before:w-full before:bg-titlebg  dark:before:bg-titlebgdark">
+                Secretaria                
+                </span>
+              </h2>
+                <div className="contact-group mb-8">
+                  <div className="mb-4">
+                    <strong>Telefone:</strong> <span><a href="tel:+351225081556">22 508 1556</a></span>
+                  </div>
+                  <div className="mb-4">
+                    <strong>Email:</strong> <span><a href="mailto:aefeup@aefeup.pt">aefeup@aefeup.pt</a></span>
```
Comment (rubuy-74) [inline components/Info/index.tsx]: I think the styling should be done via CSS.

**Review by rubuy-74 — CHANGES_REQUESTED** [review]

**Review by rubuy-74 — APPROVED** [review]
LGTM :rocket:

**Pull request merged**